FrameworkSystem/private/monitoring/ServiceInterface.py

```diff
@@ -30,15 +38,12 @@ def __createCatalog( self ):
     """
     Creates a Monitoring catalog connector
     """
-    from DIRAC.FrameworkSystem.private.monitoring.MonitoringCatalog import MonitoringCatalog
```

Comment:
Hum, you're sure about these changes ? From what I have experienced in the last weeks when fiddling with the Core for DErrno, if the import is inside a method, there is a good reason... normally late loading. Especially in a Singleton class